cmd/podman/trust.go

```diff
@@ -230,15 +215,18 @@ func setTrustCmd(c *cli.Context) error {
 	if args[0] == "default" {
 		policyContentStruct.Default = newReposContent
 	} else {
-		exists := false
+		if policyFileExists == false && len(policyContentStruct.Default) == 0 {
+			return errors.Errorf("Default trust policy must be set to create the policy file.")
+		}
```

Comment:
Does this really need to be added both here and around line 202 above? It seems to me that just having one check (probably around here here) should be sufficient. Right now `policyFileExists` more or less means “did this check already”.

Reply:
not necessary around line202, just let it return earlier.